src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  let discount = 0;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      discount = 20;
+      break;
+    case days >= 7:
+      discount = 50;
+      break;
+  }
+
+  const TOTAL_COST = days * RENT_COST - discount;
```

Comment:
The variable `TOTAL_COST` is not necessary. You can directly return the result of the calculation.